ci-operator/config/redhat-appstudio/rhtap-installer/redhat-appstudio-rhtap-installer-main.yaml

```diff
@@ -0,0 +1,49 @@
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.20
+images:
+- dockerfile_literal: |-
+    FROM src
```

Comment:
We can also have this whole Dockerfile on the main repo (rhtap-installer) itself, no?

Reply:
It will look similar to https://github.com/openshift/release/blob/master/ci-operator/config/redhat-developer/odo/redhat-developer-odo-main__odo-ocp4.13-lp-interop.yaml#L7-L9 

Also you can see their Dockerfile on the repo https://github.com/redhat-developer/odo/blob/main/openshift-ci/build-root/Dockerfile

Reply:
I would like to reuse the image that we have in pipeline-service, such as quay.io/redhat-pipeline-service/static-checks:main, but I don't know why it doesn't work in this CI. for now I am using the dockerfile_literal directly to make them work

Reply:
So is it temporary?

Reply:
yes